app/gateway/changelog.md

```diff
@@ -3182,6 +3180,80 @@ openid-connect
+**Release Date** 2023/11/28
+
+### Features
+#### Core
+* Added support for observing the time consumed by some components in the given request.
+* A unique Request ID is now populated in the error log, access log, error templates, log serializer, and in a new X-Kong-Request-Id header (configurable for upstream/downstream using the `headers` and `headers_upstream` configuration options).
+
+#### Enterprise
+* **Admin API:** Added counters such as routes, plugins, licenses, and deployment info to the report component. Also, added a checksum and timestamp to the output.
+
+#### Plugins
+* [OpenID Connect](/hub/kong-inc/openid-connect/) (`openid-connect`)
+  * New field `unauthorized_destroy_session`, which when set to true, it destroys the session (delete the user's session cookie) when the request is unauthorized. Default is set to true. Set to false to preserve the session.
+
+### Fixes
+#### Core
+* Dismissed confusing debug log from the Redis rate limiting tool.
+* Removed the asynchronous timer in `syncQuery()` to prevent hang risk.
+* Updated the DNS client to follow configured timeouts in a more predictable manner.
+* Ensured pluginserver protobuf includes are placed in the correct path in packages.
+* Added missing support for consumer group tags.
+* Fixed an issue that caused Kong Gateway to fail to start if `proxy_access_log` is `off`.
+* Removed asynchronous timer in `syncQuery()` to prevent hang risk.
+* Fixed an issue that called `store_connection` without passing `self`.
+* Now Kong Gateway uses deep copies of route, service, and consumer objects when log serialize.
+* Added per request debugging.
+* Fixed an issue that caused a failure to broadcast keyring material when using the cluster strategy.
+* Addressed a problem where an abnormal socket connection would be reused when querying the PostgreSQL database.
+* Fixed a plugin server issue that triggered invalidation when the instance was reset.
+
+#### Enterprise
+* Fixed an issue with the local variable `pkey` shadowing the package `pkey`. This caused the "attempt to call field 'new' (a nil value)" error message to display when calling `pkey.new`.
+
+#### Plugins
+* [mTLS Authentication](/hub/kong-inc/mtls-auth/) (`mtls-auth`)
+  * The plugin no longer caches the network failure when doing revocation check.
+* [AWS-Lambda](/hub/kong-inc/aws-lambda/) (`aws-lambda`)
+  * Gradually initializes AWS library on a first use to remove startup delay caused by AWS metadata discovery.
+* [OpenID Connect](/hub/kong-inc/openid-connect/) (`openid-connect`)
+  * Now allows preserving the session when there's a `401`.
+  * Fixed an issue on token revocation on logout where the code was revoking the refresh token when it was supposed to revoke the access token when using the discovered revocation endpoint.
+* Collector (`collector`)
+  * Fixed an issue where Kong cannot start after upgrading to versions greater than or equal to 2.8.4.1 because the deprecated Collector plugin was still being used.
```

Comment:
```suggestion
* Collector (`collector`)
  * Fixed an issue where Kong Gateway couldn't start after upgrading to versions greater than or equal to 2.8.4.1 because the deprecated Collector plugin was still being used.
```